deepchem/molnet/load_function/load_dataset_template.py

```diff
@@ -211,13 +201,13 @@ def load_mydataset(
       frac_valid=frac_valid,
       frac_test=frac_test)
 
-  # Check for transformers that require a dataset
-  normalize = kwargs.get("normalize", True)  # Normalization transform
-  move_mean = kwargs.get("move_mean", True)  # Zero out mean of dataset
-  if normalize:
-    transformers.append(
-        deepchem.trans.NormalizationTransformer(
-            transform_y=True, dataset=train_dataset, move_mean=move_mean))
+  # Initialize transformers
+  transformers = [
+      DEFAULT_TRANSFORMERS[t](dataset, **transformer_kwargs[t])
```

Comment:
As a slight bugfix to my earlier suggestion, we should do this as a keyword argument:
```
DEFAULT_TRANSFORMERS[t](dataset=dataset, **transformer_kwargs[t])
```

Reply:
Got it, thanks!